app/components/filter/Filter.tsx

```diff
@@ -98,27 +158,24 @@ const Filter = (props: TFilterProps) => {
               placeholder={item.title}
               options={item.options}
               isMulti={true}
-              onChange={(option: TOption[]) =>
-                onSelectChange(item.name, option)
-              }
-              selectedOptions={filterParams[item.name] || []}
+              id={item.name}
```

Comment:
there is no need to pass an argument to child component if it is not used inside of it